src/main/scala/swiss/dasch/api/ProjectsEndpoints.scala

```diff
@@ -166,6 +166,16 @@ final case class ProjectsEndpoints(base: BaseEndpoints) {
     .tag(projects)
     .description("Authorization: read:project:1234 scope required.")
 
+  val deleteProjectsErase = base.secureEndpoint.delete
```

Comment:
instead of `deleteProjectsErase` I would use `deleteProjectsPermamently`. I think we didn't agree on naming convention here yet, however my proposal sounds a bit more natural. Applying this suggestion would trigger name change for related methods.

Reply:
I am following the `methodPath` naming convention as all the other `val`s. Are you suggesting to change the path as well?

Reply:
A data recovery service blog post does indeed suggest erase is meant to signify permanent deletion, there were others. I suppose then it is good as it is.

https://www.stellarinfo.com/blog/what-is-the-difference-between-file-erase-and-file-delete/

Reply:
Thanks for the link. Whatever will remain here, it needs to be explicitly passed to the other team members to make the clear distinction between deactivation and permanent deletion of project.